.github/workflows/deploy-job.yml

```diff
@@ -1,4 +1,4 @@
-name: deploy-step
+name: deploy-job
```

Comment:
Any specific reason to rename this file. 
I agree that `deploy-job` is more suitable as a name, but we use deploy-step in every project.

Reply:
I'm not sure. This was done by the original author

Reply:
it's fine, let's keep it as it is